src/malli/clj_kondo.cljc

```diff
@@ -53,6 +53,7 @@
 (defmethod accept 'sequential? [_ _ _ _] :sequential)
 (defmethod accept 'ratio? [_ _ _ _] :int) ;;??
 (defmethod accept 'bytes? [_ _ _ _] :char-sequence) ;;??
+(defmethod accept 'ifn? [_ _ _ _] :ifn)
```

Comment:
could you please align all the `'ifn?` mappings after the `fn?` in all the relevant places.

Reply:
I've moved any `ifn?` code after corresponding `fn?` code in both the implementation and tests. If there's any more clean up you'd like me to do, happy to do so.